core/models.py

```diff
@@ -1,3 +1,13 @@
 from django.db import models
-
+from account.models import Account
 # Create your models here.
+
+
+class Notification(models.Model):
+    user = models.ForeignKey(Account, related_name="lecturer_courses", on_delete=models.SET_NULL, null=True, blank=True)
+    message = models.TextField
+    created_at = models.DateTimeField(auto_now=True)
```

Comment:
We don't want this field to be updated every time the model instance is saved, change it to auto_now_add=True.